### PR TITLE
Fix `bugs-tab` where the emoji contains a dash

### LINKED ISSUE #5068: `bugs-tab` does not appear if the bug label emoji has a `-` in it
### Please ensure:

- [X] The bug is caused by Refined GitHub. It doesn't happen if I disable the extension.

### Include in this issue:

- [ ] Screenshots/video/gif demonstrating the bug, if it’s visual
- [ ] Console errors, if any

### Description

Hi and thanks a ton for making this addon!

I manage a few repositories where I renamed the default bug label and can't find a way to make the "Bugs" tab appear in this circumstance.

Here's the thing: I like emojis and therefore like to name this category `🐞 bug`
This may be easily addressable by extending the `supportedLabels` in `source/features/bugs-tab.tsx`

Hope you guys consider this type of request a bug and not a feature request, I wasn't too sure about the issue category for this one.

### Example URL

https://github.com/zdharma-continuum/zinit

### Browser(s) used

Firefox

### PR DESCRIPTION
**Pull request by yakov116**
Fixes #5068

## Test URLs
https://github.com/zdharma-continuum/zinit

## Screenshot
image

### CODE REVIEW

**Comment by yakov116** [comment]
@refined-github/maintainers we have another issue.

Looks like GitHub is responding the an actual emoji 😢 

```graphql
{"data":
	{"repository":
		{"labels":
			{"nodes":
				[{"name":"🐞 bug","issues":{"totalCount":2}}]
			}
		}
	}
}
```

File: source/features/bugs-tab.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import SearchQuery from '../github-helpers/search-query';
 import abbreviateNumber from '../helpers/abbreviate-number';
 import {highlightTab, unhighlightTab} from '../helpers/dom-utils';
 
-const supportedLabels = /^(bug|confirmed-bug|type:bug|kind:bug|:\w+:bug)$/i;
+const supportedLabels = /^(bug|confirmed-bug|type:bug|kind:bug|(:[\w_-]+:|\p{Emoji})\s?bug)$/iu;
```
Comment (cheap-glitch) [inline source/features/bugs-tab.tsx:15]:
```suggestion
const supportedLabels = /^(bug|confirmed-bug|type:bug|kind:bug|(:[\w-]+:|\p{Emoji})bug)$/iu;
```


 * `_` is already included in `\w`
 * we don't need to acount for a space, they're removed before testing

**Review by cheap-glitch — APPROVED** [review]
Successfully tested on Firefox :sparkles: (and the suggested changes too)

**Review by fregante — APPROVED** [review]
Ohhh yes. But _ is missing?

**Comment by yakov116** [comment]
@fregante as per https://github.com/refined-github/refined-github/pull/5100#discussion_r753836190
> _ is already included in \w

**Comment by fregante** [comment]
> Looks like GitHub is responding the an actual emoji 😢

Can you open an issue? If GitHub responds with an emoji when the label is :something:, does it also work the other way? That is: "label:😃" when in reality it should be `label:":smile:"`

**Comment by yakov116** [comment]
? my regex fixed the issue

**Comment by yakov116** [comment]
Unless your saying the github will sometimes respond as an emoji and sometimes not?

My theory is that if there is a space after the emoji they return the emoji.

**Pull request merged**